### PR TITLE
Switch issues locking to github actions

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by LeviHarrison** [comment]
Although there's no practical reason, it might be nice to have it comment from @prombot instead: https://github.com/dessant/lock-threads#using-a-personal-access-token.

**Comment by roidelapluie** [comment]
It should not comment at all, because commenting is creating useless notifications (this is locking issues closed without novelty for 6+months.)

**Comment by LeviHarrison** [comment]
Yeah sorry, I meant lock.

**Comment by roidelapluie** [comment]
I'd rather not introduce a token from the robot. Every token we create is a token that can be leaked accidentally :-)

**Comment by roidelapluie** [comment]
cc @SuperQ what's your opinion on having prombot doing this?

File: .github/workflows/lock.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 4 * * *'
```
Comment (SuperQ) [inline .github/workflows/lock.yml:5]: Let's randomize the time a bit more.

**Review by SuperQ — COMMENTED** [review]
Yes, prombot is probably the right token to use.

File: .github/workflows/lock.yml (hunk 1 above)
Comment (roidelapluie) [reply .github/workflows/lock.yml]: done

**Comment by roidelapluie** [comment]
switched to prombot

File: .github/workflows/lock.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v3
+        with:
+          process-only: 'issues'
+          issue-inactive-days: '180'
+          github-token: ${{ secrets.PROMBOT_LOCKTHREADS_TOKEN}}
```
Comment (LeviHarrison) [inline .github/workflows/lock.yml:22]:
```suggestion
          github-token: ${{ secrets.PROMBOT_LOCKTHREADS_TOKEN }}
```

Comment (roidelapluie) [reply .github/workflows/lock.yml]: fixed thanks :)

**Review by LeviHarrison — APPROVED** [review]

**Pull request merged**